### PR TITLE
Add tests for example grid/dataGrids.html

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

**Comment by mcking65** [comment]
@jessebeach, @tatermelon, @sh0ji, this is ready for review.

File: test/tests/dataGrids.js [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+        ') th:nth-of-type(' + columnIndex + ')';
+    }
+
+    // If the element has "tabindex", it is the candidate for focus
+    const cellElement = document.querySelector(selector);
+    if (cellElement.hasAttribute('tabindex')) {
+      return document.activeElement === cellElement;
+    }
+
+    // Look for an interative element in the gridcell to find candidate for focu
```
Comment (jessebeach) [inline test/tests/dataGrids.js]: nit: 'focus'

File: test/tests/dataGrids.js [hunk 2]
```diff
@@ -0,0 +1,763 @@
...
+      }
+
+    }
+  }
+});
+
+ariaTest('Control+home moves focus', exampleFile, 'key-control-home', async (t) => {
+  t.plan(28);
+
+  // Examples 1 2, and 3
```
Comment (jessebeach) [inline test/tests/dataGrids.js]: Oxford comma?

File: test/tests/dataGrids.js [hunk 3]
```diff
@@ -0,0 +1,763 @@
...
+        columnIndex = 1;
+      }
+    }
+  }
+});
+
+ariaTest('Control+end moves focus', exampleFile, 'key-control-end', async (t) => {
+  t.plan(28);
+
+  // Examples 1 2, and 3
```
Comment (jessebeach) [inline test/tests/dataGrids.js]: ditto

**Pull request merged**